packages/shared/refThrottled/index.ts

```diff
@@ -10,18 +12,23 @@ import { useThrottleFn } from '../useThrottleFn'
  * @param  delay  A zero-or-greater delay in milliseconds. For event callbacks, values around 100 or 250 (or even higher) are most useful.
  * @param [trailing] if true, update the value again after the delay time is up
  * @param [leading] if true, update the value on the leading edge of the ms timeout
+ * @param [defaultClone] default clone function for non-primitive values
```

Comment:
Maybe you just named it `cloneHandler`. Cause people can provide a custom clone function, that's not default.